HealLookup.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.Generic;
```

Comment:
Holy tomatoes, Batman.  So clean, elegant, and pure.  I love it.  I wish I had had the brains to figure it out like this.  Thank you.

Reply:
You do, just gotta develop the eye and the skills.